File: 2219-MaximumNumberOfWordsFoundInSentences/2219-MaximumNumberOfWordsFoundInSentences.py
# Last updated: 6/4/2026, 9:42:13 PM
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        max_c=0
        for i  in sentences:
            a=i.split(" ")
            max_c=max(max_c,len(a))
        return max_c
        

File: 2219-MaximumNumberOfWordsFoundInSentences/2219-MaximumNumberOfWordsFoundInSentences.java
// Last updated: 6/4/2026, 9:42:19 PM
class Solution {
    public int mostWordsFound(String[] sentences) {
        int max=0;
        for( String s:sentences){
            int word=s.split(" ").length;
            max=Math.max(word,max);
        }
        return max;
    }
}